doc/source/how_to/coding/add_constraints_to_lattice.ipynb

```diff
@@ -22,24 +22,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": null,
```

Comment:
The cells need to be executed.

Reply:
Even after executing all the cells in the notebook, this variable still seems to be null for me.